### ISSUE TITLE
training loss for "score" model type

### ISSUE THREAD

**Issue by gaoyubing98** [ZhengYinan-AIR/Diffusion-Planner#38]
Thank you for your work! I encountered an issue similar to #21 that hasn't been resolved yet. When the model type is set to "score", the initial loss is very high (8–9), and after 500 epochs of training, it drops to 6. However, when the model type is set to "x_start", the initial loss is around 1. Why is this the case? Why can't the loss for the "score" type continue to decrease?

**Comment by aboutwahtcanido** [comment]
I'm facing the same problem. Has anyone made progress?

**Comment by smilebetterworld** [comment]
@gaoyubing98 @aboutwahtcanido ， hi， bros， the training loss decerease very slowly, and how do you improve it for score model type? and whether you verify it works well?

**Comment by TrayfourYu** [comment]
Please check #62 . Maybe there is a bug in the model. @gaoyubing98 @aboutwahtcanido
